### PR TITLE
done with cart & wishlist server-calls

### PR DESCRIPTION
**Pull request by Varun-28**
Done with Cart and Wishlist Server Calls

**Features:**
- Cart Server Calls
- Wishlist Server Calls
- fixed netlify redirects

**Files to review:**
- useCartServerCalls.js
- useWishlistServerCalls.js

### CODE REVIEW

**Review by shilpe26 — APPROVED** [review]
🔹Code is working as expected
🔹UI looks good
🔹Good to merge

File: src/components/CartCard.jsx [hunk 1]
```diff
@@ -1,25 +1,18 @@
 import React from "react";
 import "../pages/cart/cart.css";
 import { useCart } from "../context/cartContext/cart-context";
-import { useWishlist } from "../context/wishlistContext/wishlist-context";
+import { useCartServerCalls } from "../context/cartContext/useCartServerCalls";
+import { useWishlistServerCalls } from "../context/wishlistContext/useWishlistServerCalls";
```
Comment (dev-enforced) [inline src/components/CartCard.jsx:5]: Can use barrel exports here

**Review by MaruthiPothuganti — APPROVED** [review]
Code looks Clean. PR can be merged.

File: src/context/cartContext/useCartServerCalls.js [hunk 2]
```diff
@@ -0,0 +1,103 @@
...
+        alert.show("Added To Cart", { type: "success" });
+      }
+    } catch (err) {
+      alert.show("Server Error: Cannot be Added!", { type: "error" });
+    }
+  };
+
+  const deleteFromCart = async (productId) => {
+    try {
+      const response = await axios.delete(`/api/user/cart/${productId}`, {
```
Comment (dev-enforced) [inline src/context/cartContext/useCartServerCalls.js:57]: Can destructure the response here itself as 
```js
const {data:{cart:cartResponse}}=api call
```
For accessing nested data in objects

**Review by dev-enforced — APPROVED** [review]
The live preview of Shoebox is working as mentioned in the PR description: https://deploy-preview-11--the-shoebox.netlify.app/cart !! Good work on the UI as well as the code required for the API calls and raising alerts!!Gave a minor suggestion which you can add on later in the project

**Pull request merged**